### PR TITLE
Render the config panel based on the URL.

### PR DESCRIPTION
**Pull request by hatched**
## Done

- Update the url handling so that the config panel is rendered based on the URL and not part of the App panel.
- Fix sticky description bug from animation branch.

## QA

- Pull code
- Run dashboard
- Navigate to the config UI for an application
- Refresh the page, it should end up loading the config panel again.
- Make your browser vertically small, focus a config input and scroll the page, the config description should stick to the header.

## Details

Fixes https://github.com/canonical-web-and-design/jaas-dashboard/pull/845

### CODE REVIEW

File: src/components/SlidePanel/SlidePanel.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ function SlidePanel({
     if (
       isActive &&
       !e.target.closest(".slide-panel") &&
+      !e.target.closest('[role="row"]') &&
```
Comment (hatched) [inline src/components/SlidePanel/SlidePanel.js:41]: Drive-by to fix a bug where if you clicked on a row in the panel it would close the panel. We'll want a more robust system than this otherwise this list will continue to grow.

File: src/components/panels/ConfigPanel/ConfigPanel.tsx [hunk 2]
```diff
@@ -220,12 +224,13 @@ export default function ConfigPanel({
           </div>
           <div className="config-panel__description col-6">
             {selectedConfig ? (
-              <FadeIn isActive={true}>
-                <div className="config-panel__description-wrapper">
-                  <h4>Configuration Description</h4>
-                  <h5>{selectedConfig.name}</h5>
-                  <pre>{selectedConfig.description}</pre>
-                </div>
+              <FadeIn
```
Comment (hatched) [inline src/components/panels/ConfigPanel/ConfigPanel.tsx:227]: Drive-by to fix an issue where the description was no longer sticky when scrolling.

File: src/components/panels/ConfigPanel/ConfigPanel.tsx [hunk 3]
```diff
@@ -16,11 +16,13 @@ import boxImage from "static/images/no-config-params.svg";
 import BooleanConfig from "./BooleanConfig";
 import TextAreaConfig from "./TextAreaConfig";
 
+import { generateIconImg } from "../../../pages/Models/Details/generators";
```
Comment (barrymcgee) [inline src/components/panels/ConfigPanel/ConfigPanel.tsx:19]: This should be an absolute import.

```suggestion
import { generateIconImg } from "pages/Models/Details/generators";
```

**Review by barrymcgee — CHANGES_REQUESTED** [review]
Small change requested

**Review by barrymcgee — APPROVED** [review]
